addons/jenkins/1.x/jenkins.yaml

```diff
@@ -1,14 +1,13 @@
----
 apiVersion: kubeaddons.mesosphere.io/v1beta1
 kind: Addon
 metadata:
   name: jenkins
   labels:
     kubeaddons.mesosphere.io/name: jenkins
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "1.9.4-1"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.9.4-2"
     appversion.kubeaddons.mesosphere.io/jenkins: "1.9.4"
-    values.chart.helm.kubeaddons.mesosphere.io/jenkins: "https://raw.githubusercontent.com/helm/charts/master/stable/jenkins/values.yaml"
+    values.chart.helm.kubeaddons.mesosphere.io/jenkins: "https://raw.githubusercontent.com/helm/charts/6ec6034/stable/jenkins/values.yaml"
```

Comment:
how about using 
https://raw.githubusercontent.com/jenkinsci/helm-charts/main/charts/jenkins/values.yaml to use the values from `jenkinsci` org.

(OR https://raw.githubusercontent.com/jenkinsci/helm-charts/f251fce11b8c7dd376059732d965112e505907a3/charts/jenkins/values.yaml to pin it)